app/lib/search/index_simple.dart

```diff
@@ -702,8 +702,10 @@ class TokenIndex {
       final Map<String, double> weights =
           _inverseIds.putIfAbsent(token, () => <String, double>{});
       // on the first insert of an entry, we populate the similarity candidates
+      print('$token ${weights.isEmpty}');
```

Comment:
did you intent to leave print statements?

Reply:
Ah, sorry, leftover debug statements, removing them...